### DISCUSSION TITLE
Disable virtual desktop switching animation individually

### DISCUSSION THREAD

**Discussion by wuyilingwei** [valinet/ExplorerPatcher#2542] [Ideas]
We all know that you can turn it off in the advanced system options of the control panel, but it will also turn off all in-window animations, making it look very stiff(even WSA will follow this setting).
Is there a way to individually disable the animation of virtual desktop switching (I have wallpaper, and the original switching of Windows will be very ugly)

Windows 11 Profestional 23H2 22631.2715
Windows Pack 1000.22677.1000.0
ExplorerPatcher 22621.2428.59.1

**Comment by pyrates999** [comment]
Please update to the latest release of EP: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2506.60.1_9e7c1e6

**Reply by wuyilingwei** [reply]
I've done the update but it doesn't seem to have this feature?

**Reply by icoolqin** [reply]
For anyone looking for an alternative solution: on Windows I ended up using a small utility called **Desktop Switcher** from the Microsoft Store. It lets you move the mouse to the bottom of the screen and scroll the wheel to move between virtual desktops.  
I like this because I don’t have to cycle through desktops with repeated key presses, and the animation feels smoother than the default. Might be worth a try if you’re experimenting with ways to tweak desktop switching.

**Comment by pyrates999** [comment]
Thank you for updating.  Always update before you request a new feature to be implemented.

Please wait for a developer to respond.

**Comment by peter34245** [comment]
And has something changed in the new versions?

**Reply by pyrates999** [reply]
The release notes are shown with each new release and pre-release of what has changed.

**Comment by WouterVanGoey** [comment]
Is there a fix for this? It used to be possible with ViveTool but not anymore..

**Reply by wuyilingwei** [reply]
What is the vive command? I haven't updated 24h2 yet, maybe I can try it.
